features/missing.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Missing
+  List missing dependencies
+
+  Scenario: No dependencies
+    Given this Cask file:
+    """
+    """
+    When I run cask "missing"
+    Then I should see command output:
+    """
+    """
+    
+  Scenario: With dependencies
+    Given this Cask file:
+    """
+    (depends-on "package-a" "0.1.2")
+    (depends-on "package-b" "0.2.1")
+    
+    (development
+    (depends-on "package-c")
+    (depends-on "package-d"))
+    """
+    When I run cask "install"
+    And I run cask "missing"
+    Then I should see command output:
+    """
+    """
+      
+  Scenario: With missing dependencies
+    Given this Cask file:
+    """
+    (depends-on "package-a" "0.1.2")
+    (depends-on "package-b" "0.2.1")
+      
+    (development
+      (depends-on "package-c")
+      (depends-on "package-d"))
+    """
+    When I run cask "list"
+    Then I should see command output:
+    """
+    """
```

Comment:
I guess you don't want the empty string here!? 😄 

Also, can you add a test for a linked package.